packages/types/src/methods/send-transfer.ts

```diff
@@ -1,6 +1,7 @@
 import { DefineRpcMethod, RpcRequest, RpcResponse } from '../rpc';
 
 export interface SendTransferResponseParams {
```

Comment:
Noticed here, shouldn't the naming be `SendTransferRequestParams` (and same for others that show `Response`?

Reply:
yes it should